### ISSUE TITLE
What's the last version for windows 7?

### ISSUE THREAD

**Issue by DumbJoe** [oleksis/youtube-dl-gui#139]

**Comment by oleksis** [comment]
The latest version until today is yt-dlg v1.8.5 . For Windows you can download the [installer (MSI) from the release](https://github.com/oleksis/youtube-dl-gui/releases/download/v1.8.5/yt-dlg-20230224.1.msi) or from the [Store](https://apps.microsoft.com/store/detail/ytdlg/XP9CCFSWS911F5)

**Comment by DumbJoe** [comment]
>The latest version until today is yt-dlg v1.8.5 . For Windows you can download the [installer (MSI) from the release](https://github.com/oleksis/youtube-dl-gui/releases/download/v1.8.5/yt-dlg-20230224.1.msi) or from the [Store](https://apps.microsoft.com/store/detail/ytdlg/XP9CCFSWS911F5)

Sorry, I should have specified; let me rephrase the question: "What's the last version that still *works* or and supported on windows 7 systems?"

**Comment by oleksis** [comment]
`yt-dlg` just need Python 3 and its dependencies, pypubsub and wxPython<=4.2.1,>=4.0.7.post2 

Recommend you use Python 3 for 64 bit architecture.

**Comment by DumbJoe** [comment]
Thanks, is there a way to download pypubsub and wxPython in portable forms so can be added whenever when you're offline? Or it's impossible to add manually?

**Comment by oleksis** [comment]
Can you provide steps or logs to reproduce the behavior you want?

**Comment by DumbJoe** [comment]
Alright, if you can link me the direct download urls for pypubsub and wxPython and where to place their compiled versions as if installed with their respective python commands (but manually), that would be most appreciated.
